fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/FedoraTypesUtils.java

```diff
@@ -299,31 +301,14 @@ public static boolean isReferenceProperty(final Node node, final String property
      */
     public static Node getClosestExistingAncestor(final Session session,
                                                   final String path) throws RepositoryException {
-        final String[] pathSegments = path.replaceAll("^/+", "").replaceAll("/+$", "").split("/");
-
-        final StringBuilder existingAncestorPath = new StringBuilder(path.length());
-        existingAncestorPath.append("/");
 
-        final int len = pathSegments.length;
-        for (int i = 0; i != len; ++i) {
-            final String pathSegment = pathSegments[i];
-
-            if (session.nodeExists(existingAncestorPath.toString() + pathSegment)) {
-                // Add to existingAncestorPath  ...
-                existingAncestorPath.append(pathSegment);
-                if (i != (len - 1)) {
-                existingAncestorPath.append("/");
-                }
-            } else {
-                if (i != 0) {
-                    existingAncestorPath.deleteCharAt(existingAncestorPath.length() - 1);
-                }
-                break;
+        String potentialPath = path.startsWith("/") ? path : "/" + path;
```

Comment:
I am not sure I see the relevance of this update in this PR... but I guess it is an improvement <shrug/>